src/AppInstallerCommonCore/UserSettings.cpp

```diff
@@ -373,6 +373,11 @@ namespace AppInstaller::Settings
 
     UserSettings::UserSettings() : m_type(UserSettingsType::Default)
     {
+// When building WinGetUtil, we don't want to use user settings so we don't do anything here.
+// Some OS APIs related to architecture that we use when reading the settings are not
+// available on Windows 10 1607, which is where we use the util. We avoid compiling that

```

Comment:
>on Windows 10 1607 [](http://example.com/codeflow?start=13&length=18)

please add a note that this is the configuration for the Azure build machine which causes the errors